hvac/v1/__init__.py

```diff
@@ -251,35 +296,74 @@ def list(self, path):
         except exceptions.InvalidPath:
             return None
 
-    def write(self, path, wrap_ttl=None, **kwargs):
+    # TODO(v4.0.0): remove overload when write doesn't use args and kwargs anymore
+    @t.overload
+    def write(
+        self, path: str, write_ttl: t.Optional[str], **kwargs: t.Dict[str, t.Any]
```

Comment:
`write_ttl` should be `wrap_ttl`

Reply:
Great catch, thank you!
```suggestion
        self, path: str, wrap_ttl: t.Optional[str], **kwargs: t.Dict[str, t.Any]
```